actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-gem "pg", ">= 0.18", "< 2.0"
+gem "pg", "~> 1.1"
```

Comment:
pg-1.1 isn't really required for ActionCable. I changed it just for consistency.